lib/rgeo/impl_helper/basic_geometry_collection_methods.rb

```diff
@@ -177,6 +177,19 @@ def boundary
       def coordinates
         @elements.map(&:coordinates)
       end
+
+      def contains?(rhs)
+        unless Feature::Point === rhs
+          raise(Error::UnsupportedOperation,
+            "Method MultiPolygon#contains? is only defined for Point"
+          )
```

Comment:
```suggestion
          super
```

There may be a super method handling that